src/api/app/components/card_component.html.haml

```diff
@@ -0,0 +1,14 @@
+.col-12.col-md-6.col-lg-4
+  .card.mb-3.repository-card
+    .card-header.d-flex.justify-content-between
+      %div
+        = header
+      %div
+        = delete_button
```

Comment:
Is it possible to check the presence of the `delete_button` so we don't introduce an empty `div`?

Reply:
Good point, done. :+1: